src/actor/character/sheet.js

```diff
@@ -45,12 +46,100 @@ export class FabulaUltimaCharacterSheet extends FabulaUltimaActorSheet {
 		// actorData = this.computeSkills(actorData);
 		// actorData = this.computeEncumbrance(actorData);
 
-		return actorData;
-	}
+    if (actorData.actor.type == 'character') {
+      this._prepareItems(actorData);
+    }
+    return actorData;
+  }
 
-}
-// 	activateListeners(html) {
-// 		super.activateListeners(html);
+  /**
+   * Handle creating a new Owned Item for the actor using initial data defined in the HTML dataset
+   * @param {Event} event   The originating click event
+   * @private
+   */
+  async _onItemCreate(event) {
+    event.preventDefault();
+    const header = event.currentTarget;
+    const type = header.dataset.type;
+    const data = duplicate(header.dataset);
+    const name = `New ${type.capitalize()}`;
+    const itemData = {
+      name: name,
+      type: type,
+      data: data
+    }
+    // Remove the type from the dataset since it's in the itemData.type prop.
+    delete itemData.data["type"];
+
+    // Finally, create the item!
+    return await Item.create(itemData, {parent: this.actor});
+  }
+
+  /**
+   * Organize and classify Items for Character sheets.
+   *
+   * @param {Object} actorData The actor to prepare.
+   *
+   * @return {undefined}
+   */
+  _prepareItems(context) {
+    // Initialize containers.
+    const bonds = [];
+
+    // Iterate through items, allocating to containers
+    for (let i of context.items) {
+      i.img = i.img || DEFAULT_TOKEN;
+      // Append to gear.
+      if (i.type === 'bond') {
+        bonds.push(i);
+      }
+    }
+
+    // Assign and return
+    context.bonds = bonds;
+  }
+  /** override **/
+  activateListeners(html) {
```

Comment:
I am just going to nod and say "sounds good" to this entire function.

Reply:
FWIW, I think it was more or less modeled after https://github.com/fvtt-fria-ligan/forbidden-lands-foundry-vtt/blob/78c390afcc7eae5683e608a75578be4ed04ddb19/src/actor/character/character-sheet.js#L53